src/test/java/org/opensearch/neuralsearch/processor/TextChunkingProcessorIT.java

```diff
@@ -124,7 +123,10 @@ public void testTextChunkingProcessor_withFixedTokenLengthAlgorithmStandardToken
         try {
             createPipelineProcessor(FIXED_TOKEN_LENGTH_PIPELINE_WITH_STANDARD_TOKENIZER_NAME);
             createTextChunkingIndex(INDEX_NAME, FIXED_TOKEN_LENGTH_PIPELINE_WITH_STANDARD_TOKENIZER_NAME);
-            Exception exception = assertThrows(Exception.class, () -> ingestDocument(TEST_LONG_DOCUMENT));
+            Exception exception = assertThrows(Exception.class, () -> {
+                String document = getDocumentFromFilePath(TEST_LONG_DOCUMENT);
+                ingestDocument(INDEX_NAME, document, null);
```

Comment:
```suggestion
                ingestDocument(INDEX_NAME, document);
```

Can we have another method `ingestDocument(INDEX_NAME, document);` and use it?

Reply:
Sure, I added an overload without the last parameter to make it neater